### PR TITLE
Add Candle

### PR DESCRIPTION
**Pull request by samisbakedham**
##### Twitter Link:
https://twitter.com/candle_labs

##### Website Link:
https://candlelabs.org

##### Logo (High resolution, preferably in .svg and .png, for application on both white and black backgrounds. Will be shown with rounded borders):
https://i.ibb.co/3zyFRVD/candleenlargedicon.png

##### Current TVL:
$123,000
##### Chain:
Candle

##### Coingecko ID (so your TVL can appear on Coingecko): (https://api.coingecko.com/api/v3/coins/list)
candle

##### Coinmarketcap ID (so your TVL can appear on Coinmarketcap): (https://api.coinmarketcap.com/data-api/v3/map/all?listing_status=active,inactive,untracked&start=1&limit=10000)
18327

##### Short Description (to be shown on DefiLlama):
Building a scaleable, zero-fee network for the best

##### Token address and ticker if any:
0xbc138bD20C98186CC0342C8e380953aF0cb48BA8 - CNDL

##### Category (full list at https://defillama.com/categories) *Please choose only one:

Privacy

##### methodology (what is being counted as tvl, how is tvl being calculated):
Staked amount in UniSwap pool on Ethereum

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/candle exports TVL: 
        
 
 
    candle                    0
    ethereum                  0
    ethereum-staking          22.74 k
    candle-staking            22.74 k
    ethereum-pool2            2.383455965411655
    candle-pool2              2.383455965411655
    staking                   45.48 k
    pool2                     4.76691193082331
    
    total                    0

File: projects/candle/index.js [hunk 1]
```diff
@@ -20,6 +20,11 @@ module.exports = {
     pool2: pool2s(stakingContracts, [USDC_CNDL_UNIV2]),
     tvl: (async) => ({}),
   },
+  candle: {
+    staking: stakings(stakingContracts, CNDL),
```
Comment (waynebruce0x) [inline projects/candle/index.js:24]: can you pass the chain name ('candle') into the stakings and pool2s functions here? at the moment they're calling ethereum

Comment (waynebruce0x) [reply projects/candle/index.js:24]: we added candle in sdk 2.3.61

Comment (samisbakedham) [reply projects/candle/index.js:24]: I'll fix that thank you and yeah the protocol was added i think but not yet the chain

Comment (waynebruce0x) [reply projects/candle/index.js:24]: I mean we added candle chain in that sdk version, so if you update the sdk in your package.json then run 'node test.js projects/candle/index.js' it should work with 'candle' chain specified

Comment (samisbakedham) [reply projects/candle/index.js:24]: > I mean we added candle chain in that sdk version, so if you update the sdk in your package.json then run 'node test.js projects/candle/index.js' it should work with 'candle' chain specified

Yeah it does, but in order for Candle to show as a chain on DefiLlama we have to merge this too I think

Comment (waynebruce0x) [reply projects/candle/index.js:24]: yeah true but we need this doing first

> can you pass the chain name ('candle') into the stakings and pool2s functions here? at the moment they're calling ethereum

**Pull request merged**